### ISSUE TITLE
Sites redirected

### ISSUE THREAD

**Issue by nobodxbodon** [opencompany/awesome-open-company#61]
For example:

[**Aleph Objects**](https://www.alephobjects.com/) - redirected to lulzbot.com
[**New Vector**](https://vector.im/) - redirected to element.io

**Comment by nobodxbodon** [comment]
[Aleph Objects website](https://www.alephobjects.com/) is down with "404 page not found".
Maybe change to [LulzBot](https://lulzbot.com/)?

**Comment by nobodxbodon** [comment]
As the broken links are growing, I'll try to move the down sites to a separate table, with some history pages hopefully from internet archive.

**Comment by waldyrious** [comment]
Yeah, a separate table would make sense. What title would you suggest? "Defunct companies", perhaps?

As for the renamed entries, I wonder if we should retain a reference to the old name — e.g. "Element (formerly New Vector)"? Or do you think that would be too noisy?

**Comment by nobodxbodon** [comment]
Is there possibility that the company still exists but they forgot to redirect to the new official website, or the business turned to 100% offline? Maybe we can reach out to committers from previous PRs for confirmation.

"Element (formerly New Vector)" seems fine to me.

**Comment by waldyrious** [comment]
Yeah, I suppose we could @-mention the relevant people in this thread and give it a few weeks, or we could even go as far as digging up their emails and contacting them that way if we're feeling generous, but doing more than that would IMO be a lot of effort for something that seems unlikely.
